### PR TITLE
Fix advanced_delay

### LINKED ISSUE #135: advanced_delay: broken
The `advanced_delay` item is broken. When a Uniform jitted is specified, there is no delay at all. When a Std. Dev jitter is specified, the delay is (seemingly) infinite. An example script to illustrate the issue was posted in this discussion:

- http://forum.cogsci.nl/discussion/5033/cant-get-advanced-delay-to-work#latest

### PR DESCRIPTION
**Pull request by smathot**
This should fix #135. I haven't implemented unit tests though. @dschreij Could you do that?

### CODE REVIEW

**Comment by dschreij** [comment]
I'll do my best, but it is really a challenge to implement useful unit tests with the way OSWeb is currently structured. Everything is very intertwined and It's really difficult to pull modules apart to test them in isolation.

File: src/js/osweb/plugins/advanced_delay.js [hunk 1]
```diff
@@ -15,52 +15,34 @@ export default class AdvancedDelay extends Item {
...
    * Gaussian distribution function.
    * @param {Number} mean - The mean value.
-   * @param {Number} stdev - The standard deviation value.
+   * @param {Number} std - The standard deviation value.
    * @return {Number} - result value
    */
-  _random_gauss (mean, stdev) {
-    var y2
-    var use_last = false
-    return function () {
```
Comment (dschreij) [inline src/js/osweb/plugins/advanced_delay.js:40]: The old implementation of _random_gauss returned a function, while the new one returns a value. Are you sure this won't break things? (and yes, I know that's what the unit tests are for ;) )

**Review by dschreij — APPROVED** [review]
I tested this with the test experiment of Robbert Mijn from the forum, and slightly adapted it to also contain a delay with the std dev method. It appears to work correctly now, so this PR can be merged in my opinion.

**Comment by dschreij** [comment]
About the unit tests: I've taken a look at ways to write unit tests for the advanced delay item, and this will be quite a challenge. Sadly, OSweb does not simply pause the execution of an experiment, but relies on the (very convoluted) events.js module to (artificially) introduce delays between items. This module does some very nifty tricks to overcome the asynchronous nature of JavaScript, that makes pausing so hard. Events.js in turn relies on the ticker that is provided by Pixi.js, which is the library that takes care of the drawing, sound playback, etc. It hence relies heavily on the browser environment, and doesn't run well in node.js, which does not really provide a drawing buffer (canvas) etc. I'm especially worried that the `requestAnimationFrame` callback that Pixi relies on for its ticker function, won't be available in a Node environment.

In short, I can spend time on writing unit tests for this module, but I think it will take a while to come up with a usable result. We can also keep this in the radar and place it in the backlog, and pick it up at a later time, which has my preference.
